dbt_style_guide.md

```diff
@@ -1,237 +1,415 @@
+  ```
+
+- Joins should list the "left" table first (i.e., the table you're joining data to):
+  ```sql
+  select
     trips.*,
     drivers.rating as driver_rating,
     riders.rating as rider_rating
-
-from trips
-left join users as drivers
+  from trips
+  left join users as drivers
     on trips.driver_id = drivers.user_id
-left join users as riders
+  left join users as riders
     on trips.rider_id = riders.user_id
 
-```
+  ```
+
+- ### Example SQL
+  ```sql
+  with
+
+  my_data as (
+      select * from {{ ref('my_data') }}
+      where not is_deleted
+  ),
+
+  some_cte as (
+      select * from {{ ref('some_cte') }}
+  ),
+
+  some_cte_agg as (
+      select
+          id,
+          sum(field_4) as total_field_4,
+          max(field_5) as max_field_5
+      from some_cte
+      group by 1
+  ),
+
+  final as (
+      select [distinct]
+          my_data.field_1,
+          my_data.field_2,
+          my_data.field_3,
+
+          -- use line breaks to visually separate calculations into blocks
+          case
+              when my_data.cancellation_date is null
+                and my_data.expiration_date is not null
+                then expiration_data
+              when my_data.cancellation_date is null
+                then my_data.start_date + 7
+              else my_data.cancellation_date
+          end as cancellation_date,
+
+          some_cte_agg.total_field_4,
+          some_cte_agg.max_field_5
+      from my_data
+      left join some_cte_agg  
+          on my_data.id = some_cte_agg.id
+      where 
+        my_data.field_1 = 'abc'
+        and (
+            my_data.field_2 = 'def'
+            or my_data.field_2 = 'ghi'
+        )
+      having count(*) > 1
+  )
+
+  select * from final
+  ```
 
 ## YAML style guide
 
-* Indents should be two spaces
-* List items should be indented
-* Use a new line to separate list items that are dictionaries where appropriate
-* Lines of YAML should be no longer than 80 characters.
-
-### Example YAML
-```yaml
-version: 2
-
-models:
-  - name: events
-    columns:
-      - name: event_id
-        description: This is a unique identifier for the event
-        tests:
-          - unique
-          - not_null
-
-      - name: event_time
-        description: "When the event occurred in UTC (eg. 2018-01-01 12:00:00)"
-        tests:
-          - not_null
-
-      - name: user_id
-        description: The ID of the user who recorded the event
-        tests:
-          - not_null
-          - relationships:
-              to: ref('users')
-              field: id
-```
+- Every subdirectory contains their own `.yml` file(s) which contain configurations for the models within the subdirectory.
+
+- A separate `.yml` file is created per top-level configuration (`sources`, `models`) that applies to the models within the subdirectory. 
+
+- YAML files should be prefixed with an underscore to keep it at the top of the subdirectory.  
+  Example: `__sources_core.yml`, `_models_core.yml`

```

Comment:
```suggestion
  Example: `_core__sources.yml`, `_core__models.yml`

```

Reply:
Alternate suggestion to be in alignment with our [examples above](https://github.com/dbt-labs/corp/pull/63/files#diff-79ee8d7d102b23eb33883f979da5a6db69e0e3336e4397a5ff2174382508f342R43):
```suggestion
  Example: `_sources_core.yml`, `_models_core.yml`

```

Reply:
I think that the _<config>_<location>.yml was the convention we'd voted on previously. Will change it to that but everyone is welcome to chime in here!

Reply:
Give me your 🌶️ takes. [Commit](https://github.com/dbt-labs/corp/pull/63/commits/bb03059b4295b374d2f0bdd598960696b009b324)